packages/playground/src/App.vue

```diff
@@ -214,6 +215,15 @@ const theme = useTheme();
 const navbarConfig = ref();
 
 const hasGrid = computed(() => !!gridStore.grid);
+watch(profileManager, () => {
+  {
+    if (profileManager.profile) {
+      openProfile.value = false;
+      $router.push("/overview");
```

Comment:
let's not push this page after login. it should use the current route and if the user logged in and was on the root `/`, show this landing page.